tests/testthat/test_mlr_learners_regr_featureless.R

```diff
@@ -1,34 +1,7 @@
 context("mlr_learners_regr_featureless")
 
-test_that("Simple training/predict", {
-  task = mlr_tasks$get("bh")
-  learner = mlr_learners$get("regr.featureless")
-  expect_learner(learner, task)
-
-  e = Experiment$new(task, learner)
-  e$train()
-  expect_class(e$model, "featureless")
-  expect_numeric(e$model, len = 2L, any.missing = FALSE)
-  e$predict()
-  e$data$prediction
-  e$prediction
-
-  expect_numeric(e$prediction$response, any.missing = FALSE, len = length(e$test_set))
-  e$score()
-  expect_number(e$performance)
-})
-
-test_that("Predict with se", {
-  task = mlr_tasks$get("bh")
-  learner = mlr_learners$get("regr.featureless")
-  learner$predict_type = "se"
-  expect_learner(learner, task)
-
-  e = Experiment$new(task, learner)$train()$predict()
-  expect_numeric(e$data$prediction$se, len = task$nrow, any.missing = FALSE, lower = 0)
-})
-
```

Comment:
Why have you removed these tests? Can you add them again?

Reply:
I thought we wanted to streamline the unit tests with an auto test for each learner. Everythin, what is tested here (simple train predict + predict type "se") is still tested with expect_autotest().

Reply:
Right, except these line:
```r
expect_class(e$model, "featureless")
expect_numeric(e$model, len = 2L, any.missing = FALSE)
```
But we can live without these tests.